index.html

```diff
@@ -33,9 +33,17 @@
   <body>
     <div class="app-wrap">
       <header class="app-header">
-        <a href="#" class="welcome">Welcome</a>
+        <a href="#" id="welcome" class="welcome">Welcome</a>
+        <div id="app-header-menu" class="app-header-menu">
+          <ul>
+            <li><a href="#work-intro-info" class="nav-link">Portfolio</a></li>
+            <li><a href="#about-me-info" class="nav-link">About</a></li>
+            <li><a href="#contact-form-info" class="nav-link">Contact</a></li>
+          </ul>
+        </div>
         <nav>
```

Comment:
- [ ] Great job so far :clap: however the mobile menu should overfly the home section and not be displayed before the home :point_down: design  [template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

![dis](https://user-images.githubusercontent.com/57562869/151015103-f4f4c554-8c4a-4a5c-9eb1-8a6b4d916d38.png)